certif/tests/unit/services/session_test.js

```diff
@@ -94,8 +94,7 @@ module('Unit | Service | session', function (hooks) {
           service.handleLocale({ isFranceDomain, localeFromQueryParam, userLocale });
 
           // then
-          sinon.assert.calledWith(localeService.setLocaleCookie, FRENCH_FRANCE_LOCALE);
-          assert.ok(true);
+          assert.true(localeService.setLocaleCookie.calledWith(FRENCH_FRANCE_LOCALE));
```

Comment:
Pourrait-on réécrire ce message de commit en : 
"""
prefer assert.true(condition)
instead of 'sinon.assert.*' and 'assert.ok(true)'
"""